From: Marc Kleine-Budde <mkl@pengutronix.de>
Date: Thu, 28 May 2015 11:03:40 +0200
Subject: [PATCH] xattr: increase MAX_IPC_BUFFER_SIZE to 1024

The program "evmctl" from the ima-evm-utils creates during "sign" operation
quite large extended attributes. For example a sha256 "security.ima" attributes
is 521 bytes long.

This patch increases the MAX_IPC_BUFFER_SIZE to 1024 to fix the problem.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 message.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/message.h b/message.h
index 0b64fb98bf94..9dfce9446df1 100644
--- a/message.h
+++ b/message.h
@@ -64,7 +64,7 @@ struct fakestat {
 #pragma pack()
 #endif
 
-#define MAX_IPC_BUFFER_SIZE 256
+#define MAX_IPC_BUFFER_SIZE 1024
 
 #if __SUNPRO_C
 #pragma pack(4)
